### ISSUE TITLE
browser compatiblity

### ISSUE THREAD

**Issue by albanm** [boundlessgeo/ol-mapbox-style#34]
Hello,

I think it would be interesting to add in the documentation a note about browser compatibility.

Can you confirm that using this module it is possible to render full mapbox styles without depending on webgl and therefore support much older browsers than mapbox-gl-js ?

If so I think it is the great strength of this module as it provides an alternative to webgl and server side rendering.

**Comment by ahocevar** [comment]
Yes, ol-mapbox-style uses OpenLayers with the canvas rendering. So no WebGL required. Feel free to submit a pull request that adds a note about this to the readme.
